components/support/nimbus-fml/src/parser.rs

```diff
@@ -2,10 +2,7 @@
 * License, v. 2.0. If a copy of the MPL was not distributed with this
```

Comment:
This file is the most interesting: we're merging the import blocks that are found in included files.